api/v1alpha1/hostedcluster_types.go

```diff
@@ -844,6 +844,10 @@ type AWSPlatformSpec struct {
 	// +kubebuilder:default=Public
 	// +optional
 	EndpointAccess AWSEndpointAccessType `json:"endpointAccess,omitempty"`
+
+	// DefaultNodepoolArch specifies the default nodepool AMI.
+	// +optional
+	DefaultNodepoolAMI string `json:"defaultnodepoolami"`
```

Comment:
We don't want to expose this in the API. This is a detail inferred from .release.